builder/vmware/common/driver_fusion6.go

```diff
@@ -124,6 +125,24 @@ func (d *Fusion6Driver) Verify() error {
 	return compareVersions(matches[1], VMWARE_FUSION_VERSION, "Fusion Professional")
 }
 
+func (d *Fusion6Driver) ToolsIsoPath(k string) string {
+	// Fusion 13.x.x changes tools iso location
+	vmxpath := filepath.Join(d.AppPath, "Contents", "Library", "vmware-vmx")
+	var stderr bytes.Buffer
+	cmd := exec.Command(vmxpath, "-v")
+	cmd.Stderr = &stderr
+	versionRe := regexp.MustCompile(`(?i)VMware [a-z0-9-]+ (\d+)\.`)
+	matches := versionRe.FindStringSubmatch(stderr.String())
+	if matches[1] < "13" {
+		return filepath.Join(d.AppPath, "Contents", "Library", "isoimages", k+".iso")
+	}
+	if runtime.GOOS == "windows" && runtime.GOARCH == "arm64" {
```

Comment:
Since this is meant to be the guest os, we can use the value of [ToolsUploadFlavor](https://developer.hashicorp.com/packer/plugins/builders/vmware/iso#tools_upload_flavor), which is what is passed into this function. The allowed values are "darwin", "linux", and "windows". 
```suggestion
	if k == "windows" && runtime.GOARCH == "arm64" {
```